src/block-components/image/style.js

```diff
@@ -35,9 +35,16 @@ const Styles = props => {
 		heightUnitCallback = null,
 		dependencies = [],
 	} = props
-
 	return (
 		<>
+			<BlockCss
+				{ ...propsToPass }
+				selector={ selector }
+				styleRule="aspect-ratio"
```

Comment:
@Arukuen this should be in camel case
```suggestion
				styleRule="aspectRatio"
```